autoscheduler/frontend/src/tests/Meeting.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import * as React from 'react';
+
+import MeetingCard from '../components/MeetingCard';
+import { Meeting, MeetingType, Section } from '../types';
+
+const testSection: Section = {
+  id: 123456,
+  subject: 'CSCE',
+  courseNum: 121,
+  sectionNum: 200,
+  minCredits: 3,
+  maxCredits: null,
+  currentEnrollment: 0,
+  instructor: 123456,
+};
+
+const testMeeting: Meeting = {
+  id: 12345,
+  crn: 123456,
+  building: 'HRBB',
+  meetingDays: new Array(7).fill(true),
+  startTimeHours: 8,
+  startTimeMinutes: 0,
+  endTimeHours: 18,
+  endTimeMinutes: 50,
+  meetingType: MeetingType.LEC,
+  section: testSection,
+};
+
+test('accepts meeting as props', () => {
+  const { container } = render(<MeetingCard meeting={testMeeting} />);
+  expect(container).toBeTruthy();
+});
+
+test('displays subject and course number', () => {
+  const { getByText } = render(<MeetingCard meeting={testMeeting} />);
+  expect(getByText(/CSCE/)).toBeTruthy();
+  expect(getByText(/121/)).toBeTruthy();
+});
+
+test('subject and course number can adapt to random values', () => {
+  // arrange
+  const randSubject = new Array(5).reduce((acc) => acc + String.fromCharCode(
```

Comment:
> I'm not sure I like using random values for unit testing, unless we give a seed to the random function (we'd have to use seedrandom in npm or something similar), since it makes tests unpredictable. There isn't a reason that I can think of that using random values would change anything versus just giving it values we made manually, either, since it's just a static length string and a number 100-599

Yeah I agree, there really isn't a logical reason to have it be a random value, since there isn't logic related to the actual value of the subject, as long as it's a certain length. See [Unit Testing Standards](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/wiki/Unit-Testing-Standards), specifically the "Readable" section.